Products/CMFPlomino/PlominoForm.py

```diff
@@ -664,6 +778,80 @@ def displayDocument(self, doc, editmode=False, creation=False,
                 parent_form_ids.append(parent_form_id)
                 request.set('parent_form_ids', parent_form_ids)
 
+        # If this is a multi page form, return the current page
+        if self.getIsMulti():
```

Comment:
Maybe we could move this section in a specific method so `displayDocument` does not get to fat.